string name
sensor_msgs/Image layer
float64 max_val
float64 min_val

